app/models/techhire_location.rb

```diff
@@ -0,0 +1,15 @@
+class TechhireLocation < ActiveRecord::Base
+  include ActiveModel::Dirty
+
```

Comment:
Mr. Fancy Pants!  ActiveModel::Dirty monitoring the state of objects!  :facepunch: !!!! Pow Pow!!!!